telemetry/telemetrybuffer.go

```diff
@@ -464,3 +466,36 @@ func ReadConfigFile(filePath string) (TelemetryConfig, error) {
 
 	return config, err
 }
+
+// ConnectToTelemetryService - Attempt to spawn telemetry process if it's not already running.
+func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, telemetryWaitTimeInMilliseconds int) {
+	var path string
+	var args []string
+
+	path = fmt.Sprintf("%v/%v", CniInstallDir, TelemetryServiceProcessName)
```

Comment:
can we move this to separate function? getTelemetryServiceDirectory() should return the directory path

Reply:
separated